### PR TITLE
97 well contentview mui

### PR DESCRIPTION
**Pull request by marmid74**
Generic table component ContentTable to be used by the ...View objects.
View objects (well,wellbore,++) needs to specify following properties using the ContentTable

-  a **tableKeys** object specifying the columns to be rendered in the table 
- send the **data** to be displayed
- if it should be possible to select single or multiple rows : **multiSelect** (true/false)

In this version WellsListView is used as parent of rendering ContentTable
Current styling of ContentTable is default "out of the box" from  material ui

### CODE REVIEW

File: Src/WitsmlExplorer.Web/ClientApp/components/PropertyViews/ContentTable.tsx [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+            <TableSortLabel
+              active={orderBy === column.id}
+              direction={order}
+              onClick={createSortHandler(column.id)}
+            >
+              {column.label}
+              {orderBy === column.id ? (
+                <span>
+                  {order === 'desc' ? 'sorted descending' : 'sorted ascending'}
+                </span>
```
Comment (AtleH) [inline Src/WitsmlExplorer.Web/ClientApp/components/PropertyViews/ContentTable.tsx]: Jeg tror dette kan slås sammen til `<span/>`

File: Src/WitsmlExplorer.Web/ClientApp/components/PropertyViews/ContentTable.tsx [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+        ))}
+      </TableRow>
+    </TableHead>
+  );
+}
+
+interface ContentTableProps {
+  tableKeys: any;
+  data: any[];
+  multiSelect: boolean;
```
Comment (AtleH) [inline Src/WitsmlExplorer.Web/ClientApp/components/PropertyViews/ContentTable.tsx]: Kan denne være default `false`?

File: Src/WitsmlExplorer.Web/ClientApp/components/PropertyViews/ContentTable.tsx [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+              active={orderBy === column.id}
+              direction={order}
+              onClick={createSortHandler(column.id)}
+            >
+              {column.label}
+              {orderBy === column.id ? (
+                <span>
+                  {order === 'desc' ? 'sorted descending' : 'sorted ascending'}
+                </span>
+              ) : null}
```
Comment (AtleH) [inline Src/WitsmlExplorer.Web/ClientApp/components/PropertyViews/ContentTable.tsx]: Jeg tror hele denne greia kan fjernes (men har ikke testet veeeeldig nøye....)

Comment (marmid74) [reply Src/WitsmlExplorer.Web/ClientApp/components/PropertyViews/ContentTable.tsx]: Ser slikt ut ja...Fjernet

**Pull request closed without merging**